### DISCUSSION TITLE
迎戰 AI 時代：企業如何藉由培訓確保人才不被淘汰

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2054] [General]
https://finance.technews.tw/2024/01/05/corporate-training-prevents-talents-from-being-replaced-by-ai/

在AI快速發展的時代下，所有人都在思考如何「不被AI取代」。工作任務有高度例行性和重複性的產業容易由AI代勞，但直接裁員並非最佳解方，而是要「以培訓取代淘汰」、「創建鼓勵持續發展與創新的學習環境和文化」，不但企業能持續進步，也能保留及重新配置人才。

我認為不只是未來透過公司培訓，現在的我們，就得開始練習持續學習，不管是AI科技、第二外語等各種新技能；也要練習能觀察社會趨勢的眼光與創新力，在這過程中，若能運用AI加強學習成效的話當然更好，因為，「AI 不會取代人類，而是『不會使用AI』的人」而且，會善用AI 的人只會更勝一籌。

**Comment by JoannaLai909** [comment]
我同意你的看法，我認為不論未來AI會發展到甚麼程度，還是會有一些方面是科技無法完成，需要依靠人類主觀情感。所以企業可以引進AI，但不能因為AI就拋棄人力資源，而是培養員工運用科技的能力，當然不願意學習的人可想而知遲早會被時代淘汰

**Comment by hunteryang0108** [comment]
我認為這樣的看法很好，AI可能不會直接取代人類，但是會使用AI的人能輕易地取代那些不會使用AI的人。為了不成為企業中被淘汰的那個人，我們應該在我們學習中融入AI的使用，善用這些工具進行高效率工作，並時刻保持學習的心態，隨時追上發展時代的腳步。
